app/views/meeting.status.html

```diff
@@ -8,7 +8,7 @@
                     </button>
                 </div>
                 <div ng-if="menuFull()" class="col-sm-4">
-                    <button type="button" class="btn db-btn-hdr glyphicon center glyphicon-home" ng-click="msc.goHome()">
+                    <button db-confirm confirm-active="msc.isEditing" confirm-config="msc.unsavedConfig" type="button" class="btn db-btn-hdr glyphicon center glyphicon-home" ng-click="msc.goHome()">
```

Comment:
Maybe rename directive attribute `confirm-active` into  `confirm-enabled`, but that doesn't block this PR.